packages/quantic/scripts/build/deploy-community.ts

```diff
@@ -312,7 +312,9 @@ async function deleteScratchOrg(
 
   try {
     let communityUrl = '';
-    runner.add(async (log) => await authorizeDevOrg(log, options));
+    if (process.env.CI) {
+      runner.add(async (log) => await authorizeDevOrg(log, options));
+    }
```

Comment:
in line `323` we re do the same step:
```
runner.add(async (log) => await authorizeDevOrg(log, options));
```
I don't think it's necessary, is it?

Reply:
Yeah same you're right.